src/components/util/misc.js

```diff
@@ -55,3 +55,7 @@ isSelectAction.selectionActionMap = {
     "multiselect" : true
 };
 
+export function isPrimitive(value) {
+    const t = typeof value;
+    return t === 'string' || t === 'number' || t === 'bigint' || t === 'boolean' || t === 'symbol' || t === 'undefined' || t === 'null';
```

Comment:
The 'type' of `null` is Object interestingly enough; if want to test for it might be better to do `t === null` ("is actual value of it null").

Reply:
@alexkb0009  Aha! Good catch, thanks. My bad, I implemented it while I was reading the MDN article (https://developer.mozilla.org/en-US/docs/Glossary/Primitive), and didn't notice the single quotes.